### DISCUSSION TITLE
fibonacci numbers

### DISCUSSION THREAD

**Discussion by fedaabdullah** [community/community#18564] [Projects and Issues] [answered]
<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <title></title>
  </head>
  <body>

    <script>

        var number = 8;
      var num1 =0 ,num2 = 1;
      var next_term =0;
      for( i=1; i< number; i ++){
        if (i <= 1 )
        next_term =i ;
       else{
        next_term = num1 +num2;
        num1 = num2;
        num2= next_term;
}
console.log("Adding" +num1 +"and" +next_term + "=" +(num1 +num2)+ "<br");
}
      </script>

  </body>
</html>

**Comment by fedaabdullah** [comment, marked as answer]
natchi.html:22 Adding1and1=2<br
natchi.html:22 Adding1and2=3<br
natchi.html:22 Adding2and3=5<br
natchi.html:22 Adding3and5=8<br
natchi.html:22 Adding5and8=13<br
natchi.html:22 Adding8and13=21<br

**Comment by Zerotask** [comment]
What's the exact problem or intent?
